db/migrate/20200603175713_add_incident_involved_a_van_to_incident_reports.rb

```diff
@@ -1,5 +0,0 @@
-class AddIncidentInvolvedAVanToIncidentReports < ActiveRecord::Migration[5.2]
-  def change
-    add_column :incident_reports, :incident_involved_a_van, :boolean, null: false
-  end
-end
```

Comment:
Deleting migrations does not remove columns from the database.

Reply:
I had to change the default value of 7 columns, so I created migrations to remove them. I then added them back in with new migrations with the same name as the old ones and set the defaults before migrating. In hind sight I should have kept the migrations, but in the moments i was getting super annoyed at seeing a bunch of new migrations with similar names.

Reply:
As we talked about in Zoom, changing existing migrations will only hurt. But we also talked about how there's a method to change the default value of the columns, so your problem of repetition is solved anyway.